src/Plugins/PublishRequestModel.cs

```diff
@@ -32,14 +32,14 @@ public enum PublishType
         Zoomin,
 
         /// <summary>
-        /// Contains a value that indicates publishing a DocBook Pdf.
+        /// Contains a value that indicates publishing to DocBook.
         /// </summary>
-        DocBookPdf,
+        DocBook,
 
         /// <summary>
         /// Contains a value that indicates publishing to AntennaHouse PDF.
         /// </summary>
-        AntennaHousePdf
+        AntennaHousePdf,
```

Comment:
Ignore Stylecop on these. It's a bug I believe in their rules/parser.

Reply:
will do